Signal/src/textsecure/Messages/TSMessagesManager.m

```diff
@@ -233,7 +233,7 @@ -(void)handleSendToMyself:(TSOutgoingMessage*)outgoingMessage {
     [self.dbConnection readWriteWithBlock:^(YapDatabaseReadWriteTransaction *transaction) {
         TSContactThread *cThread = [TSContactThread getOrCreateThreadWithContactId:[SignalKeyingStorage.localNumber toE164] transaction:transaction];
         [cThread saveWithTransaction:transaction];
-        TSIncomingMessage *incomingMessage = [[TSIncomingMessage alloc] initWithTimestamp:(outgoingMessage.timeStamp + 1) inThread:cThread messageBody:outgoingMessage.body attachments:outgoingMessage.attachments];
+        TSIncomingMessage *incomingMessage = [[TSIncomingMessage alloc] initWithTimestamp:(outgoingMessage.timestamp + 1) inThread:cThread messageBody:outgoingMessage.body attachments:outgoingMessage.attachments];
```

Comment:
Timestamp is a word, not the camel case of two words, so I thought we would stick to that.
http://en.wikipedia.org/wiki/Timestamp